js/script.js

```diff
@@ -787,10 +787,27 @@ function Add_EventHandlers_To_Document()
     })
 }
 
+function selectPalette() {
+    var nes = document.getElementById("palette-div");
+    var gameboy = document.getElementById("gameboy-palette-div");
+    var radioNES = document.getElementById("radioNES");
+    var radioGameboy =document.getElementById("radioGB");
+
+    if (radioNES.checked) {
+        nes.style.display = "block";
```

Comment:
I think it's best to use classes here, as you are breaking DRY.

Maybe we can just keep as a constant JS object as well. WDYT?

Reply:
I think because it is a small function, a class might not fit very well. Perhaps implementing a object with these at the top would be better for eliminating redundancies. I'll look into doing that and run it by you to confirm.

Reply:
Totally agree!

And I should have clarified, I did not mean a JavaScript class, but rather employing CSS class:

eg.
```css 
.checked-style {
  display: block;
  ....
}
```

WDYT of this?